MANUAL_REGRESSION.md

```diff
@@ -538,6 +538,24 @@ Given user opened the link with `?uploadFallback=false` flag
 5. Click on "Try the basic camera mode instead"
     - You should be able to take a picture with your native camera
 
+### 33c. Live document capture facingMode: {exact: "environment"} constraint
+(mobile browsers, and browsers on Microsoft Surface or other hybrid device)
```

Comment:
Would be good add iPad to the list so we can tests different behaviours between tablets and hybrid devices.